kube/services/revproxy/00nginx-config.yaml

```diff
@@ -236,5 +242,35 @@ data:
               send_timeout 300;
               proxy_pass http://sheepdog-service/;
           }
+          location /workspace/ {
+              auth_request /auth-proxy;
+              auth_request_set $remoteUser $upstream_http_REMOTE_USER;
+              proxy_set_header REMOTE_USER $remoteUser;
```

Comment:
What is $upstream_http_REMOTE_USER - could you put some comments here explaining what's going on?